android-app/app/build.gradle

```diff
@@ -46,6 +46,7 @@ android {
     }
     buildFeatures {
         viewBinding true
+        buildConfig = true
```

Comment:
```suggestion
        buildConfig true
```

we don't use the "assignment" syntax anywhere in this file, could we use the "declarative" approach consistently?

Reply:
yes it works. Good point, I didn't test it